install_config/web_console_customization.adoc

```diff
@@ -227,6 +227,7 @@ angular
       .when('/about', {
         templateUrl: 'extensions/about/about.html',
         controller: 'AboutController'
+        hawtioPluginLoader.addModule('aboutPageExtension');
```

Comment:
sorry wasnt clear, should be the very last line in the whole code block, after the last `);`